src/NHibernate.Validator.Tests/NHibernate.Validator.Tests.csproj

```diff
@@ -490,6 +493,9 @@
     <EmbeddedResource Include="Specifics\NHV25\Show.nhv.xml" />
     <EmbeddedResource Include="Specifics\NHV25\Tv.nhv.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
```

Comment:
Looks like an undue change.

Reply:
Removed. It was done by VS automatically.